components/director/internal/open_resource_discovery/apiclient/config.go

```diff
@@ -0,0 +1,12 @@
+package apiclient
+
+import (
+	"time"
+)
+
+// OrdAggregatorClientConfig is the configuration needed for ord aggregator client
+type OrdAggregatorClientConfig struct {
+	ClientTimeout             time.Duration `envconfig:"default=30s"`
```

Comment:
There is no way to set different timeout than 30s. 
Is this intentional? - if so probably is better ti use constant instead configuration.